core/finality-grandpa/primitives/src/lib.rs

```diff
@@ -154,24 +161,51 @@ pub const PENDING_CHANGE_CALL: &str = "grandpa_pending_change";
 /// WASM function call to get current GRANDPA authorities.
 pub const AUTHORITIES_CALL: &str = "grandpa_authorities";
 
-decl_runtime_apis! {
```

Comment:
We will need to re-add the runtime API, it's required for exposing equivocation reporting functionality (#3868). But I can re-add it in my PR after this is merged.